content/en/tracing/trace_search_and_analytics/agent_trace_search.md

```diff
@@ -52,6 +50,15 @@ DD_APM_ANALYZED_SPANS="<SERVICE_NAME_1>|<OPERATION_NAME_1>=1,<SERVICE_NAME_2>|<O
 {{% /tab %}}
 {{< /tabs >}}
 
+In Datadog, every automatically instrumented service has an `<OPERATION_NAME>`, which is used to set the type of request being traced. For example, if you're tracing a Python Flask application, you might have a `flask.request` as your operation name. In a Node application using Express, you would have `express.request` ask your operation name.
```

Comment:
Afaik Flash is Python-only, so why mention it?

Reply:
It's an example because we never explain what the operation name is across the doc.
BUT we should document the operation name soon (this or next sprint) this will be removed then since a language specific example doesn't really make sense i agree.

Reply:
🆗